template/setup.py

```diff
@@ -0,0 +1,27 @@
+from setuptools import setup, find_packages
+
+project_name = 'YOUR-PACKAGE'
+package_name = 'YOUR_PACKAGE'
+
+setup(
+    name=project_name,
+    version='0.1',
+    long_description=open('README.md').read(),
+    url='https://github.com/BrewBlox/' + project_name,
+    author='BrewPi',
```

Comment:
replace by placeholders. Add test to check that they have changed from defaults. <-- applies to other required changes too.